velox/docs/functions/spark/math.rst

```diff
@@ -240,10 +240,10 @@ Mathematical Functions
     If ``x`` contains non-hexadecimal character, the function returns NULL.
     When ``x`` contains even number of characters, each pair is converted to a single byte. The number of bytes in the result is half the number of bytes in the input.
     When ``x`` contains a single character, the result contains a single byte whose value matches the hexadecimal character.
-    When ``x`` contains an odd number characters greater than 2, the first character is ignored, the remaining pairs of characters are converted to bytes, then zero byte is added at the end of the output. ::
+    When ``x`` contains an odd number characters greater than 2, the first character is decoded to one byte and put at index 0 of the output, the remaining pairs of characters are converted to bytes, and put start from index 1 of the output. ::
```

Comment:
nit: perhaps,

When x contains an odd number of characters greater than 2, the first character is decoded into the first byte of the result and the remaining pairs of characters are decoded into subsequent bytes.

Reply:
thanks, already apply this better one.